test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -483,15 +483,20 @@ private String summarizeProcessInstanceSubscription(final Record<?> record) {
   private String summarizeVariable(final Record<?> record) {
     final var value = (VariableRecordValue) record.getValue();
 
-    return new StringBuilder()
+    final var builder = new StringBuilder();
+    builder
         .append(value.getName())
         .append("->")
         .append(value.getValue())
         .append(" in <process ")
         .append("[")
         .append(shortenKey(value.getProcessInstanceKey()))
-        .append("]>")
-        .toString();
+        .append("]");
+    if (value.getProcessInstanceKey() != value.getScopeKey()) {
+      // only add if they're different, no need to state things twice
+      builder.append(" at [").append(shortenKey(value.getScopeKey()));
+    }
+    return builder.append("]>").toString();
```

Comment:
If the variable has the process instance as the scope then the formatted message looks not symmetric. There is an additional square bracket.

`x -> 1 in <process[1]]>`

Reply:
Fixed, I've also applied `String.format` to make the formatting more readable.